packages/latte-phpstan-compiler/src/LatteToPhpCompiler.php

```diff
@@ -111,7 +113,11 @@ private function decorateStmts(array $phpStmts, array $componentNamesAndTypes):
 
         $nClassNodeVisitor = new NClassNodeVisitor();
 
-        $linkNodeVisitor = new LinkNodeVisitor($this->simpleNameResolver, $this->nodeValueResolver);
+        $linkNodeVisitor = new LinkNodeVisitor(
+            $this->simpleNameResolver,
+            $this->nodeValueResolver,
+            $this->presenterFactoryFaker->getPresenterFactory()
+        );
```

Comment:
This is probably result of forgotten service registration.
Could you move this to neon config, so we avoid manual maintenance of constructor?

Reply:
Do you mean to create LinkNodeVisitor as a service? All NodeVisitors above are created manually.

Reply:
Yes. They should be service instead. We miss Rector rule to spot them :)

Reply:
maybe it's because one or more use arguments from method decorateStmts()... OK I'll create LinkNodeVisitor as a service

Reply:
I've created an issue for this https://github.com/symplify/symplify/issues/3815. I'll try to prepare PR as next step